src/common/plugin/makeCurrencyTools.ts

```diff
@@ -167,8 +168,20 @@ export function makeCurrencyTools(
         : publicAddress
     },
 
-    getSplittableTypes(_walletInfo: EdgeWalletInfo): string[] {
-      return []
+    getSplittableTypes(walletInfo: EdgeWalletInfo): string[] {
+      const getFormatsForNetwork = (network: string): string[] => {
```

Comment:
move 'getFormatsForNetwork' to the plugin's utils file

Reply:
Done